bettermdptools/algorithms/rl.py

```diff
@@ -31,8 +31,9 @@
 
 
 class RL:
-    def __init__(self, env):
+    def __init__(self, env, seed=None):
```

Comment:
Actually I think we should just import the seed from seed.py so users don't have to keep passing it around in multiple places